pontoon/base/models.py

```diff
@@ -1371,6 +1371,32 @@ def find_and_replace(self, find, replace, user):
         Translation.objects.bulk_create(translations_to_create)
         return translations
 
+    def authors(self):
+        """
+        Return a QuerySet of translation authors.
+        """
+        return User.objects.filter(translation__in=self).distinct()
+
+    def counts_per_minute(self):
+        """
+        Return a dictionary of translation counts per minute.
+        """
+        translations = (
+            self
+            .extra({'minute': "date_trunc('minute', date)"})
+            .order_by('minute')
+            .values('minute')
+            .annotate(count=Count('id'))
+        )
+
+        data = []
+        for period in translations:
+            data.append([
+                utils.convert_to_unix_time(period['minute']),
```

Comment:
I think we could replace this loop with time conversion inside the sql query and  .values_list() call at the end.